src/main/java/com/google/cloud/tools/gradle/endpoints/framework/server/task/GenerateClientLibsTask.java

```diff
@@ -81,8 +81,7 @@ public void setServiceClasses(List<String> serviceClasses) {
   @TaskAction
   void generateClientLibs() throws Exception {
 
-    getProject().delete(clientLibDir);
-    clientLibDir.mkdirs();
+    // This action intentionally does not clean the output directory
```

Comment:
I guess, I mean if the comment you put needs to be there, an explanation would be helpful.